### DISCUSSION TITLE
Floating ScrollArea -> Now problem with plot after solve that

### DISCUSSION THREAD

**Discussion by Igor636965736c610a** [emilk/egui#3568] [Q&A] [answered]
I have a problem that I can't set the slider so that when I hover over it, it slides out like on the website https://www.egui.rs. My slider defaults to Type: Solid and I want to set it to Floating and generally I can't find all the settings that are on the website in the "Details" section. I will also show my code and toml dependencies if they may be useful.

![image](https://github.com/emilk/egui/assets/102369546/4777691c-dd84-4976-b6cc-ea8c9a11d9e1)
This is what i want:
![image](https://github.com/emilk/egui/assets/102369546/7d0544f5-435c-447f-b447-978549926598)
my code:
![image](https://github.com/emilk/egui/assets/102369546/5b246e0a-bdfd-4521-8dc4-c17ca60c41e1)

and this is with not ScrollingArea
![image](https://github.com/emilk/egui/assets/102369546/5d069479-2f07-4619-8f70-095b15164cb8)

and Toml:
![image](https://github.com/emilk/egui/assets/102369546/2ebe3bc8-b5a6-47d4-bc26-e5157abddd4f)

**Comment by YgorSouza** [comment]
The UI you see in the demo app is here:

https://github.com/emilk/egui/blob/beb4714e406adfaad54eb7365785d9a4ba90f789/crates/egui/src/style.rs#L497-L508

And what you want to put in your code is this:

```rs
ui.spacing_mut().scroll = ScrollStyle::floating();
```

However, this is already the default, as you can see here:

https://github.com/emilk/egui/blob/beb4714e406adfaad54eb7365785d9a4ba90f789/crates/egui/src/style.rs#L421-L425

So the problem is actually that this feature hasn't been released yet. It was merged only 9 days ago, in  #3539, and will be in the upcoming 0.24 release. So if you want to use it now, you will have to add egui and eframe as git dependencies instead of using the latest release from crates.io.

**Reply by Igor636965736c610a** [reply]
I did as you wrote. It worked, but now a new problem has arisen haha. Because I had to update egui to version 0.23, the default chart display settings have changed and it doesn't look like it's impossible to change where the "axes" should be displayed from .show_axes([false, true]) where I set them to were in the middle of the chart

![image](https://github.com/emilk/egui/assets/102369546/da594c7c-3534-4fc5-8b99-8551b04e4045)

and it was like this before in 0.22
![image](https://github.com/emilk/egui/assets/102369546/e00e6bb0-1e02-4b90-910e-f4578ed33027)

and 0.22 plot is not exist ;/ 
![image](https://github.com/emilk/egui/assets/102369546/f236e9cd-d298-4740-a1a3-27aad175c566)

**Reply by YgorSouza** [reply, marked as answer]
Yes, there is a discussion about that on #3437. The plot API still needs some work, I guess.

**Reply by Igor636965736c610a** [reply]
Thank you so much!
